Representable/representable
# test community size while drawing

**changes**
- small change adds a size test while drawing (rather than on submit) to warn users if they are selecting a community that is too large

**to test**
- go to entry page
- try to select community over 800 block groups in size
- check warning message shows, no js error, etc

**screenshot**
![image](https://user-images.githubusercontent.com/41943646/89955145-b689f400-dc75-11ea-9a42-1bdda252d426.png)
